packages/react-composites/src/composites/CallComposite/adapter/CallingSoundSubscriber.ts

```diff
@@ -22,11 +22,11 @@ const CALL_TRANSFER_SUBCODE = 7015;
  * @private
  */
 export class CallingSoundSubscriber {
-  private call: CallCommon;
+  private call?: CallCommon;
   private soundsLoaded?: CallingSoundsLoaded;
   private callee: CommunicationIdentifier[] | undefined;
 
-  constructor(call: CallCommon, callee?: CommunicationIdentifier[], sounds?: CallingSounds) {
+  constructor(call?: CallCommon, callee?: CommunicationIdentifier[], sounds?: CallingSounds) {
```

Comment:
can this be legitimately constructed without a call object?
Looks like the subscriber will no-op throughout (i.e. do nothing) if call is undefined, in  which case we should expect a valid call object and only call the subscriber if there is a call